2017-12-03T21:00:40.242+0900 I CONTROL  [initandlisten] MongoDB starting : pid=9419 port=37018 dbpath=/data/shard0/rs1 64-bit host=kris-ThinkPad-X1-Carbon-5th
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] db version v3.4.10
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] git version: 078f28920cb24de0dd479b5ea6c66c644f6326e9
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] allocator: tcmalloc
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] modules: none
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] build environment:
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     distmod: ubuntu1604
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     distarch: x86_64
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten]     target_arch: x86_64
2017-12-03T21:00:40.243+0900 I CONTROL  [initandlisten] options: { net: { port: 37018 }, processManagement: { fork: true }, replication: { replSet: "s0" }, sharding: { clusterRole: "shardsvr" }, storage: { dbPath: "/data/shard0/rs1" }, systemLog: { destination: "file", path: "s0-r1.log" } }
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] 
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2017-12-03T21:00:40.266+0900 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=7269M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),checkpoint=(wait=60,log_size=2GB),statistics_log=(wait=0),
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: You are running this process as the root user, which is not recommended.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] ** WARNING: /sys/kernel/mm/transparent_hugepage/enabled is 'always'.
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] **        We suggest setting it to 'never'
2017-12-03T21:00:40.354+0900 I CONTROL  [initandlisten] 
2017-12-03T21:00:40.354+0900 W SHARDING [initandlisten] Started with --shardsvr, but no shardIdentity document was found on disk in admin.system.version. This most likely means this server has not yet been added to a sharded cluster.
2017-12-03T21:00:40.378+0900 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/shard0/rs1/diagnostic.data'
2017-12-03T21:00:40.401+0900 I REPL     [initandlisten] Did not find local voted for document at startup.
2017-12-03T21:00:40.401+0900 I REPL     [initandlisten] Did not find local replica set configuration document at startup;  NoMatchingDocument: Did not find replica set configuration document in local.system.replset
2017-12-03T21:00:40.401+0900 I NETWORK  [thread1] waiting for connections on port 37018
2017-12-03T21:00:45.646+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36582 #1 (1 connection now open)
2017-12-03T21:00:45.647+0900 I -        [conn1] end connection 127.0.0.1:36582 (1 connection now open)
2017-12-03T21:00:45.647+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36586 #2 (1 connection now open)
2017-12-03T21:00:45.647+0900 I NETWORK  [conn2] received client metadata from 127.0.0.1:36586 conn2: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:45.647+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-03T21:00:45.648+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-03T21:00:47.651+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36600 #3 (2 connections now open)
2017-12-03T21:00:47.652+0900 I -        [conn3] end connection 127.0.0.1:36600 (2 connections now open)
2017-12-03T21:00:47.681+0900 I REPL     [replExecDBWorker-0] Starting replication storage threads
2017-12-03T21:00:47.760+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36604 #4 (2 connections now open)
2017-12-03T21:00:47.761+0900 I NETWORK  [conn4] received client metadata from 127.0.0.1:36604 conn4: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:00:47.762+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-03T21:00:47.762+0900 I REPL     [replication-0] Starting initial sync (attempt 1 of 10)
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] New replica set config in use: { _id: "s0", version: 1, protocolVersion: 1, members: [ { _id: 0, host: "localhost:37017", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 1, host: "localhost:37018", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "localhost:37019", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: 60000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5a23e76d05bd90c23620c9e4') } }
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] This node is localhost:37018 in the config
2017-12-03T21:00:47.762+0900 I REPL     [ReplicationExecutor] transition to STARTUP2
2017-12-03T21:00:47.763+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-03T21:00:47.763+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 1ms (2 connections now open to localhost:37019)
2017-12-03T21:00:47.764+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-03T21:00:47.764+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state STARTUP2
2017-12-03T21:00:47.764+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 1ms (2 connections now open to localhost:37019)
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] sync source candidate: localhost:37017
2017-12-03T21:00:47.805+0900 I STORAGE  [replication-0] dropAllDatabasesExceptLocal 1
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] ******
2017-12-03T21:00:47.805+0900 I REPL     [replication-0] creating replication oplog of size: 6161MB...
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] Starting WiredTigerRecordStoreThread local.oplog.rs
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] The size storer reports that the oplog contains 0 records totaling to 0 bytes
2017-12-03T21:00:47.825+0900 I STORAGE  [replication-0] Scanning the oplog to determine where to place markers for truncation
2017-12-03T21:00:47.922+0900 I REPL     [replication-0] ******
2017-12-03T21:00:47.922+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:47.923+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-03T21:00:47.924+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:47.925+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (2 connections now open to localhost:37017)
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] No need to apply operations. (currently at { : Timestamp 1512302445000|1 })
2017-12-03T21:00:47.926+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending connection to host localhost:37017 due to bad connection status; 1 connections to that host remain open
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Finished fetching oplog during initial sync: CallbackCanceled: Callback canceled. Last fetched optime and hash: { ts: Timestamp 1512302445000|1, t: -1 }[-3972036409932239766]
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Initial sync attempt finishing up.
2017-12-03T21:00:47.926+0900 I REPL     [replication-1] Initial Sync Attempt Statistics: { failedInitialSyncAttempts: 0, maxFailedInitialSyncAttempts: 10, initialSyncStart: new Date(1512302447762), initialSyncAttempts: [], fetchedMissingDocs: 0, appliedOps: 0, initialSyncOplogStart: Timestamp 1512302445000|1, initialSyncOplogEnd: Timestamp 1512302445000|1, databases: { databasesCloned: 0 } }
2017-12-03T21:00:47.930+0900 I REPL     [replication-0] initial sync done; took 0s.
2017-12-03T21:00:47.930+0900 I REPL     [replication-0] Starting replication fetcher thread
2017-12-03T21:00:47.931+0900 I REPL     [replication-0] Starting replication applier thread
2017-12-03T21:00:47.931+0900 I REPL     [replication-0] Starting replication reporter thread
2017-12-03T21:00:47.931+0900 I REPL     [rsSync] transition to RECOVERING
2017-12-03T21:00:47.931+0900 I REPL     [rsBackgroundSync] could not find member to sync from
2017-12-03T21:00:47.931+0900 I REPL     [ReplicationExecutor] Member localhost:37019 is now in state SECONDARY
2017-12-03T21:00:47.931+0900 I REPL     [rsSync] transition to SECONDARY
2017-12-03T21:00:57.933+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-03T21:00:58.933+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-03T21:00:58.936+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:00:58.937+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (2 connections now open to localhost:37017)
2017-12-03T21:01:47.764+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37019 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:02:10.569+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36776 #5 (3 connections now open)
2017-12-03T21:02:10.569+0900 I NETWORK  [conn5] received client metadata from 127.0.0.1:36776 conn5: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.648+0900 I INDEX    [repl writer worker 1] build index on: admin.system.version properties: { v: 2, key: { version: 1 }, name: "incompatible_with_version_32", ns: "admin.system.version" }
2017-12-03T21:02:10.648+0900 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:10.650+0900 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:10.653+0900 I COMMAND  [repl writer worker 4] setting featureCompatibilityVersion to 3.4
2017-12-03T21:02:10.659+0900 I SHARDING [repl writer worker 8] initializing sharding state with: { _id: "shardIdentity", configsvrConnectionString: "csReplSet/localhost:57040,localhost:57041,localhost:57042", shardName: "s0", clusterId: ObjectId('5a23e786bf51d32cbcc946a4') }
2017-12-03T21:02:10.659+0900 I SHARDING [repl writer worker 8] first cluster operation detected, adding sharding hook to enable versioning and authentication to remote servers
2017-12-03T21:02:10.660+0900 I NETWORK  [repl writer worker 8] Starting new replica set monitor for csReplSet/localhost:57040,localhost:57041,localhost:57042
2017-12-03T21:02:10.661+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-03T21:02:10.661+0900 I SHARDING [thread2] creating distributed lock ping thread for process kris-ThinkPad-X1-Carbon-5th:37018:1512302530:-6460390088971249845 (sleeping for 30000ms)
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.662+0900 I NETWORK  [replSetDistLockPinger] Successfully connected to localhost:57042 (1 connections now open to localhost:57042 with a 5 second timeout)
2017-12-03T21:02:10.662+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-03T21:02:10.662+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 0ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.663+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 1ms (2 connections now open to localhost:57040)
2017-12-03T21:02:10.672+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: LockStateChangeFailed: findAndModify query predicate didn't match any lock document
2017-12-03T21:02:10.740+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36824 #6 (4 connections now open)
2017-12-03T21:02:10.740+0900 I NETWORK  [conn6] received client metadata from 127.0.0.1:36824 conn6: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.821+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36850 #7 (5 connections now open)
2017-12-03T21:02:10.822+0900 I NETWORK  [conn7] received client metadata from 127.0.0.1:36850 conn7: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.837+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36876 #8 (6 connections now open)
2017-12-03T21:02:10.837+0900 I NETWORK  [conn8] received client metadata from 127.0.0.1:36876 conn8: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.856+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36894 #9 (7 connections now open)
2017-12-03T21:02:10.856+0900 I NETWORK  [conn9] received client metadata from 127.0.0.1:36894 conn9: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.929+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36946 #10 (8 connections now open)
2017-12-03T21:02:10.929+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36948 #11 (9 connections now open)
2017-12-03T21:02:10.929+0900 I NETWORK  [conn10] received client metadata from 127.0.0.1:36946 conn10: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.929+0900 I NETWORK  [conn11] received client metadata from 127.0.0.1:36948 conn11: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:10.991+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36974 #12 (10 connections now open)
2017-12-03T21:02:10.991+0900 I NETWORK  [conn12] received client metadata from 127.0.0.1:36974 conn12: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:11.140+0900 I INDEX    [repl writer worker 10] build index on: school.students properties: { v: 2, key: { student_id: 1.0 }, name: "student_id_1", ns: "school.students" }
2017-12-03T21:02:11.140+0900 I INDEX    [repl writer worker 10] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2017-12-03T21:02:11.141+0900 I INDEX    [repl writer worker 10] build index done.  scanned 0 total records. 0 secs
2017-12-03T21:02:11.200+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:36992 #13 (11 connections now open)
2017-12-03T21:02:11.201+0900 I NETWORK  [conn13] received client metadata from 127.0.0.1:36992 conn13: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.450+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37024 #14 (12 connections now open)
2017-12-03T21:02:27.451+0900 I NETWORK  [conn14] received client metadata from 127.0.0.1:37024 conn14: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:27.606+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37042 #15 (13 connections now open)
2017-12-03T21:02:27.606+0900 I NETWORK  [conn15] received client metadata from 127.0.0.1:37042 conn15: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.664+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:02:40.666+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-03T21:02:40.667+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37062 #16 (14 connections now open)
2017-12-03T21:02:40.667+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s0/localhost:37017,localhost:37018,localhost:37019
2017-12-03T21:02:40.667+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s1/localhost:47017,localhost:47018,localhost:47019
2017-12-03T21:02:40.668+0900 I NETWORK  [shard registry reload] Starting new replica set monitor for s2/localhost:57017,localhost:57018,localhost:57019
2017-12-03T21:02:40.667+0900 I NETWORK  [conn16] received client metadata from 127.0.0.1:37062 conn16: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.670+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-03T21:02:40.671+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:37068 #17 (15 connections now open)
2017-12-03T21:02:40.672+0900 I NETWORK  [conn17] received client metadata from 127.0.0.1:37068 conn17: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-03T21:02:40.672+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-03T21:02:40.675+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37019 (1 connections now open to localhost:37019 with a 5 second timeout)
2017-12-03T21:02:40.679+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-03T21:02:40.681+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47018 (1 connections now open to localhost:47018 with a 5 second timeout)
2017-12-03T21:02:40.683+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-03T21:02:40.684+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-03T21:02:40.685+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57018 (1 connections now open to localhost:57018 with a 5 second timeout)
2017-12-03T21:02:40.686+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-03T21:03:10.663+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Ending idle connection to host localhost:57040 because the pool meets constraints; 1 connections to that host remain open
2017-12-03T21:08:40.678+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:08:40.680+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-03T21:12:40.690+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:12:40.691+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-03T21:36:49.550+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-03T21:36:49.558+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 8ms (3 connections now open to localhost:37017)
2017-12-03T21:38:53.578+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-03T21:41:10.760+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-03T21:41:10.760+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-03T21:51:40.784+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T21:51:40.785+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-03T22:28:10.868+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T22:28:10.871+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 4ms (1 connections now open to localhost:57041)
2017-12-03T23:04:10.945+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:04:10.947+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 3ms (1 connections now open to localhost:57041)
2017-12-03T23:10:40.960+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-03T23:10:40.962+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T00:01:11.089+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T00:01:11.091+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T00:14:11.120+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T00:14:11.122+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T01:24:41.293+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T01:24:41.294+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T01:29:11.304+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T01:29:11.306+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T02:19:41.439+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T02:19:41.441+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T02:37:41.489+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T02:37:41.491+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T02:43:11.504+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T02:43:11.506+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T03:13:11.584+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T03:13:11.586+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T03:55:11.692+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T03:55:11.694+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T04:08:41.726+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T04:08:41.727+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 1ms (1 connections now open to localhost:57041)
2017-12-04T04:09:41.729+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T04:09:41.730+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T04:14:41.741+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T04:14:41.743+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T05:46:41.981+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:46:41.982+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T05:58:42.030+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T05:58:42.031+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T06:19:23.833+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T06:19:23.834+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (3 connections now open to localhost:37017)
2017-12-04T06:20:23.834+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T06:36:42.117+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:36:42.118+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T06:49:12.149+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T06:49:12.151+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 2ms (1 connections now open to localhost:57042)
2017-12-04T06:51:23.900+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T06:51:23.901+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (3 connections now open to localhost:37017)
2017-12-04T06:52:23.900+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T08:13:42.369+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T08:13:42.373+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T15:41:54.175+0900 I REPL     [ReplicationExecutor] Starting an election, since we've seen no PRIMARY in the past 10000ms
2017-12-04T15:41:54.190+0900 I REPL     [ReplicationExecutor] conducting a dry run election to see if we could be elected
2017-12-04T15:41:54.191+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.191+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37019
2017-12-04T15:41:54.194+0900 I REPL     [ReplicationExecutor] VoteRequester(term 1 dry run) received a yes vote from localhost:37019; response message: { term: 1, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.194+0900 I REPL     [ReplicationExecutor] dry election run succeeded, running for election
2017-12-04T15:41:54.196+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] VoteRequester(term 2) received a yes vote from localhost:37019; response message: { term: 2, voteGranted: true, reason: "", ok: 1.0 }
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] election succeeded, assuming primary role in term 2
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] transition to PRIMARY
2017-12-04T15:41:54.198+0900 I REPL     [ReplicationExecutor] Entering primary catch-up mode.
2017-12-04T15:41:54.198+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending connection to host localhost:37017 due to bad connection status; 2 connections to that host remain open
2017-12-04T15:41:54.199+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Connecting to localhost:37017
2017-12-04T15:41:54.211+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57040 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.211+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T15:41:54.212+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37019, took 21ms (2 connections now open to localhost:37019)
2017-12-04T15:41:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 24ms (3 connections now open to localhost:37017)
2017-12-04T15:41:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 19ms (3 connections now open to localhost:37017)
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state SECONDARY
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Caught up to the latest optime known via heartbeats after becoming primary.
2017-12-04T15:41:54.216+0900 I REPL     [ReplicationExecutor] Exited primary catch-up mode.
2017-12-04T15:41:54.217+0900 I REPL     [rsBackgroundSync] Replication producer stopped after oplog fetcher finished returning a batch from our sync source.  Abandoning this batch of oplog entries and re-evaluating our sync source.
2017-12-04T15:41:54.217+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Successfully connected to localhost:37017, took 18ms (3 connections now open to localhost:37017)
2017-12-04T15:41:55.216+0900 I REPL     [rsSync] transition to primary complete; database writes are now permitted
2017-12-04T15:41:57.019+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:41:57.019+0900 I REPL     [SyncSourceFeedback] SyncSourceFeedback error sending update to localhost:37017: InvalidSyncSource: Sync source was cleared. Was localhost:37017
2017-12-04T15:41:57.178+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40968 #18 (16 connections now open)
2017-12-04T15:41:57.179+0900 I NETWORK  [conn18] received client metadata from 127.0.0.1:40968 conn18: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:57.182+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40970 #19 (17 connections now open)
2017-12-04T15:41:57.183+0900 I NETWORK  [conn19] received client metadata from 127.0.0.1:40970 conn19: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:59.216+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40996 #20 (18 connections now open)
2017-12-04T15:41:59.216+0900 I NETWORK  [conn20] received client metadata from 127.0.0.1:40996 conn20: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:41:59.217+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:40998 #21 (19 connections now open)
2017-12-04T15:41:59.217+0900 I NETWORK  [conn21] received client metadata from 127.0.0.1:40998 conn21: { driver: { name: "NetworkInterfaceASIO-RS", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:07.237+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:41006 #22 (20 connections now open)
2017-12-04T15:42:07.237+0900 I NETWORK  [conn22] received client metadata from 127.0.0.1:41006 conn22: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T15:42:10.310+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T15:42:10.310+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 0ms (1 connections now open to localhost:57042)
2017-12-04T15:42:10.586+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57040)
2017-12-04T15:42:10.588+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57040 (1 connections now open to localhost:57040 with a 5 second timeout)
2017-12-04T15:42:10.590+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:37017)
2017-12-04T15:42:10.592+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37017 (1 connections now open to localhost:37017 with a 5 second timeout)
2017-12-04T15:42:10.593+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:47017)
2017-12-04T15:42:10.595+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47017 (1 connections now open to localhost:47017 with a 5 second timeout)
2017-12-04T15:42:10.596+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 16 secs, remote host 127.0.0.1:57017)
2017-12-04T15:42:10.598+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57017 (1 connections now open to localhost:57017 with a 5 second timeout)
2017-12-04T15:42:24.212+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T15:42:24.214+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T15:42:54.201+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37019 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T15:42:54.215+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 2 connections to that host remain open
2017-12-04T15:42:54.218+0900 I ASIO     [NetworkInterfaceASIO-Replication-0] Ending idle connection to host localhost:37017 because the pool meets constraints; 1 connections to that host remain open
2017-12-04T16:21:41.131+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:21:41.135+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 4ms (1 connections now open to localhost:57042)
2017-12-04T16:45:25.185+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58774 #23 (21 connections now open)
2017-12-04T16:45:25.185+0900 I NETWORK  [conn23] received client metadata from 127.0.0.1:58774 conn23: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:25.192+0900 I REPL     [conn23] stepping down from primary, because a new term has begun: 3
2017-12-04T16:45:25.192+0900 I REPL     [replExecDBWorker-2] transition to SECONDARY
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] legacy transport layer closing all connections
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 23
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 22
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 21
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 20
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 19
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 18
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 4
2017-12-04T16:45:25.192+0900 I NETWORK  [replExecDBWorker-2] Skip closing connection for connection # 2
2017-12-04T16:45:25.192+0900 I -        [conn16] end connection 127.0.0.1:37062 (21 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn10] end connection 127.0.0.1:36946 (20 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn9] end connection 127.0.0.1:36894 (19 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn14] end connection 127.0.0.1:37024 (18 connections now open)
2017-12-04T16:45:25.192+0900 I -        [conn15] end connection 127.0.0.1:37042 (17 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn17] end connection 127.0.0.1:37068 (16 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn11] end connection 127.0.0.1:36948 (15 connections now open)
2017-12-04T16:45:25.193+0900 I -        [conn7] end connection 127.0.0.1:36850 (14 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn13] end connection 127.0.0.1:36992 (13 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn12] end connection 127.0.0.1:36974 (12 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn8] end connection 127.0.0.1:36876 (11 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn6] end connection 127.0.0.1:36824 (10 connections now open)
2017-12-04T16:45:25.197+0900 I -        [conn5] end connection 127.0.0.1:36776 (9 connections now open)
2017-12-04T16:45:25.203+0900 I COMMAND  [conn20] command local.oplog.rs command: getMore { getMore: 38453556961, collection: "oplog.rs", maxTimeMS: 5000, term: 2, lastKnownCommittedOpTime: { ts: Timestamp 1512372445000|1, t: 2 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342829000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 2, readConcern: { afterOpTime: { ts: Timestamp 1512342829000|1, t: 1 } } } planSummary: COLLSCAN cursorid:38453556961 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 }, acquireWaitCount: { r: 1 }, timeAcquiringMicros: { r: 320 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 1074905ms
2017-12-04T16:45:25.204+0900 I -        [conn4] end connection 127.0.0.1:36604 (8 connections now open)
2017-12-04T16:45:25.204+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58790 #24 (8 connections now open)
2017-12-04T16:45:25.205+0900 I NETWORK  [conn24] received client metadata from 127.0.0.1:58790 conn24: { driver: { name: "NetworkInterfaceASIO-Replication", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:25.205+0900 I -        [conn23] end connection 127.0.0.1:58774 (8 connections now open)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:37018)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:47019)
2017-12-04T16:45:25.208+0900 I NETWORK  [PeriodicTaskRunner] Socket closed remotely, no longer connected (idle 1094 secs, remote host 127.0.0.1:57019)
2017-12-04T16:45:25.208+0900 I COMMAND  [conn18] command local.oplog.rs command: getMore { getMore: 35231168519, collection: "oplog.rs", maxTimeMS: 5000, term: 2, lastKnownCommittedOpTime: { ts: Timestamp 1512372445000|1, t: 2 } } originatingCommand: { find: "oplog.rs", filter: { ts: { $gte: Timestamp 1512342829000|1 } }, tailable: true, oplogReplay: true, awaitData: true, maxTimeMS: 60000, term: 2, readConcern: { afterOpTime: { ts: Timestamp 1512342829000|1, t: 1 } } } planSummary: COLLSCAN cursorid:35231168519 keysExamined:0 docsExamined:0 numYields:0 nreturned:0 reslen:563 locks:{ Global: { acquireCount: { r: 4 }, acquireWaitCount: { r: 1 }, timeAcquiringMicros: { r: 5332 } }, Database: { acquireCount: { r: 2 } }, oplog: { acquireCount: { r: 2 } } } protocol:op_command 1074910ms
2017-12-04T16:45:25.208+0900 I -        [conn18] end connection 127.0.0.1:40968 (7 connections now open)
2017-12-04T16:45:25.213+0900 I REPL     [ReplicationExecutor] Member localhost:37017 is now in state PRIMARY
2017-12-04T16:45:25.256+0900 I NETWORK  [replSetDistLockPinger] Marking host localhost:57041 as failed :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.256+0900 W SHARDING [replSetDistLockPinger] pinging failed for distributed lock pinger :: caused by :: PrimarySteppedDown: Primary stepped down while waiting for replication. 
2017-12-04T16:45:25.892+0900 I -        [conn21] end connection 127.0.0.1:40998 (6 connections now open)
2017-12-04T16:45:31.178+0900 I REPL     [rsBackgroundSync] sync source candidate: localhost:37017
2017-12-04T16:45:31.178+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T16:45:31.179+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 1ms (1 connections now open to localhost:37017)
2017-12-04T16:45:31.186+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Connecting to localhost:37017
2017-12-04T16:45:31.188+0900 I ASIO     [NetworkInterfaceASIO-RS-0] Successfully connected to localhost:37017, took 2ms (2 connections now open to localhost:37017)
2017-12-04T16:45:31.202+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58872 #25 (6 connections now open)
2017-12-04T16:45:31.203+0900 I NETWORK  [conn25] received client metadata from 127.0.0.1:58872 conn25: { driver: { name: "NetworkInterfaceASIO-ShardRegistry", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:31.740+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57042
2017-12-04T16:45:31.741+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57042, took 1ms (1 connections now open to localhost:57042)
2017-12-04T16:45:32.011+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58910 #26 (7 connections now open)
2017-12-04T16:45:32.011+0900 I NETWORK  [conn26] received client metadata from 127.0.0.1:58910 conn26: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.026+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Socket closed remotely, no longer connected (idle 7 secs, remote host 127.0.0.1:57041)
2017-12-04T16:45:32.026+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58920 #27 (8 connections now open)
2017-12-04T16:45:32.027+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57041 (1 connections now open to localhost:57041 with a 5 second timeout)
2017-12-04T16:45:32.027+0900 I NETWORK  [conn27] received client metadata from 127.0.0.1:58920 conn27: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.028+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58922 #28 (9 connections now open)
2017-12-04T16:45:32.028+0900 I NETWORK  [conn28] received client metadata from 127.0.0.1:58922 conn28: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.028+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58924 #29 (10 connections now open)
2017-12-04T16:45:32.028+0900 I NETWORK  [conn29] received client metadata from 127.0.0.1:58924 conn29: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.029+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:37018 (1 connections now open to localhost:37018 with a 5 second timeout)
2017-12-04T16:45:32.030+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:47019 (1 connections now open to localhost:47019 with a 5 second timeout)
2017-12-04T16:45:32.030+0900 W NETWORK  [ReplicaSetMonitor-TaskExecutor-0] No primary detected for set s1
2017-12-04T16:45:32.031+0900 I NETWORK  [ReplicaSetMonitor-TaskExecutor-0] Successfully connected to localhost:57019 (1 connections now open to localhost:57019 with a 5 second timeout)
2017-12-04T16:45:32.309+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58960 #30 (11 connections now open)
2017-12-04T16:45:32.309+0900 I NETWORK  [conn30] received client metadata from 127.0.0.1:58960 conn30: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.309+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58964 #31 (12 connections now open)
2017-12-04T16:45:32.310+0900 I NETWORK  [conn31] received client metadata from 127.0.0.1:58964 conn31: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.312+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58966 #32 (13 connections now open)
2017-12-04T16:45:32.312+0900 I NETWORK  [conn32] received client metadata from 127.0.0.1:58966 conn32: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.337+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58984 #33 (14 connections now open)
2017-12-04T16:45:32.338+0900 I NETWORK  [conn33] received client metadata from 127.0.0.1:58984 conn33: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.340+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58990 #34 (15 connections now open)
2017-12-04T16:45:32.341+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:58994 #35 (16 connections now open)
2017-12-04T16:45:32.341+0900 I NETWORK  [conn34] received client metadata from 127.0.0.1:58990 conn34: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:32.342+0900 I NETWORK  [conn35] received client metadata from 127.0.0.1:58994 conn35: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:48.933+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59142 #36 (17 connections now open)
2017-12-04T16:45:48.934+0900 I NETWORK  [conn36] received client metadata from 127.0.0.1:59142 conn36: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.058+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59146 #37 (18 connections now open)
2017-12-04T16:45:49.059+0900 I NETWORK  [conn37] received client metadata from 127.0.0.1:59146 conn37: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:45:49.455+0900 I NETWORK  [thread1] connection accepted from 127.0.0.1:59162 #38 (19 connections now open)
2017-12-04T16:45:49.455+0900 I NETWORK  [conn38] received client metadata from 127.0.0.1:59162 conn38: { driver: { name: "MongoDB Internal Client", version: "3.4.10" }, os: { type: "Linux", name: "LinuxMint", architecture: "x86_64", version: "18.3" } }
2017-12-04T16:48:01.746+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T16:48:01.748+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
2017-12-04T16:50:25.205+0900 I -        [conn19] end connection 127.0.0.1:40970 (19 connections now open)
2017-12-04T16:50:25.205+0900 I -        [conn20] end connection 127.0.0.1:40996 (18 connections now open)
2017-12-04T16:50:25.216+0900 I -        [conn22] end connection 127.0.0.1:41006 (17 connections now open)
2017-12-04T16:50:41.223+0900 I -        [conn25] end connection 127.0.0.1:58872 (16 connections now open)
2017-12-04T17:13:31.812+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57041
2017-12-04T17:13:31.814+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57041, took 2ms (1 connections now open to localhost:57041)
2017-12-04T18:02:32.917+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Connecting to localhost:57040
2017-12-04T18:02:32.919+0900 I ASIO     [NetworkInterfaceASIO-ShardRegistry-0] Successfully connected to localhost:57040, took 2ms (1 connections now open to localhost:57040)
